src/helpers/messages.js

```diff
@@ -70,11 +70,16 @@ function getHeader(scope) {
  * @returns {String} Returns the message text
  */
 function getActionMessage(scope, submitButtonLabel = __('OK')) {
+    var msg;
     switch (scope) {
         case 'section':
         case 'testSection':
         case 'part':
-            return `${__('Click "%s" to continue', submitButtonLabel)}.`;
+            msg = __('Click "%s" to continue', submitButtonLabel);
+            if (msg.length > 1) {
```

Comment:
Indeed, this condition won't work as expected.
Moreover, it will partially address the issue.
I'd suggest adding a `trim()`

```suggestion
            msg = __('Click "%s" to continue', submitButtonLabel).trim();
            if (msg) {
```